services/metadata/src/metadata/routes/domain/permanent_id_requests.clj

```diff
@@ -0,0 +1,80 @@
+(ns metadata.routes.domain.permanent-id-requests
+  (:use [common-swagger-api.schema :only [describe
+                                          PagingParams
+                                          SortFieldDocs
+                                          SortFieldOptionalKey
+                                          StandardUserQueryParams]]
+        [clojure-commons.error-codes]
+        [metadata.routes.domain.common])
+  (:require [metadata.persistence.permanent-id-requests :as db]
+            [schema.core :as s])
+  (:import [java.util UUID]))
+
+(def PermanentIDRequestIdParam (describe UUID "The Permanent ID Requests's UUID"))
+(def PermanentIDRequestTypeEnum (apply s/enum (map :type (db/list-permanent-id-request-types))))
```

Comment:
I'm not really worried about linters and compilers, but I was a little bit worried about the namespace being loaded directly. Upon closer inspection, it appears that `metadata.core` explicitly loads the namespace after the configuration file has been loaded, though. Sorry for the false alarm.